pkg/oonf_api/0001-add-RIOT-support.patch

```diff
@@ -133,9 +133,7 @@ index dedab2c..fdc3e82 100644
 +#ifndef RIOT_VERSION
  #include <net/if.h>
 +#else
-+#include "net_help.h"
-+#include "inet_ntop.h"
-+#include "inet_pton.h"
```

Comment:
I did not test it either, but judging from the diff it looked like those were only included for the `AF_` macros.